### ISSUE TITLE
virtualenv issue on python 3.4

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#5380]
1.1-maint branch:

```
+python -m pip install virtualenv
Collecting virtualenv
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/7e/d0/8209bd40c9e1910a0b95bcc77f207e35b5b3657beb7f65d18434f9be22be/virtualenv-20.0.32-py2.py3-none-any.whl (4.9MB)
    100% |████████████████████████████████| 4.9MB 341kB/s 
Requirement already satisfied: appdirs<2,>=1.4.3 in /home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages (from virtualenv)
Collecting filelock<4,>=3.0.0 (from virtualenv)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/93/83/71a2ee6158bb9f39a90c0dea1637f81d5eef866e188e1971a1b1ab01a35a/filelock-3.0.12-py3-none-any.whl
Requirement already satisfied: six<2,>=1.9.0 in /home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages (from virtualenv)
Collecting distlib<1,>=0.3.1 (from virtualenv)
  Using cached https://files.pythonhosted.org/packages/f5/0a/490fa011d699bb5a5f3a0cf57de82237f52a6db9d40f33c53b2736c9a1f9/distlib-0.3.1-py2.py3-none-any.whl
Collecting importlib-metadata<3,>=0.12; python_version < "3.8" (from virtualenv)
  Using cached https://files.pythonhosted.org/packages/5f/bb/47f6789a2433e2ae73b7d4c657082fba84945e6af754a99b5e33980ad65a/importlib_metadata-1.1.3-py2.py3-none-any.whl
Collecting importlib-resources>=1.0; python_version < "3.7" (from virtualenv)
  Using cached https://files.pythonhosted.org/packages/2f/f7/b4aa02cdd3ee7ebba375969d77c00826aa15c5db84247d23c89522dccbfa/importlib_resources-1.0.2-py2.py3-none-any.whl
Collecting zipp>=0.5 (from importlib-metadata<3,>=0.12; python_version < "3.8"->virtualenv)
  Using cached https://files.pythonhosted.org/packages/96/0a/67556e9b7782df7118c1f49bdc494da5e5e429c93aa77965f33e81287c8c/zipp-1.2.0-py2.py3-none-any.whl
Collecting pathlib2; python_version == "3.4.*" or python_version < "3" (from importlib-metadata<3,>=0.12; python_version < "3.8"->virtualenv)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/e9/45/9c82d3666af4ef9f221cbb954e1d77ddbb513faf552aea6df5f37f1a4859/pathlib2-2.3.5-py2.py3-none-any.whl
Collecting typing; python_version < "3.5" (from importlib-resources>=1.0; python_version < "3.7"->virtualenv)
Collecting scandir; python_version < "3.5" (from pathlib2; python_version == "3.4.*" or python_version < "3"->importlib-metadata<3,>=0.12; python_version < "3.8"->virtualenv)
  Cache entry deserialization failed, entry ignored
Installing collected packages: filelock, distlib, zipp, scandir, pathlib2, importlib-metadata, typing, importlib-resources, virtualenv
Successfully installed distlib-0.3.1 filelock-3.0.12 importlib-metadata-1.1.3 importlib-resources-1.0.2 pathlib2-2.3.5 scandir-1.10.0 typing-3.7.4.3 virtualenv-20.0.32 zipp-1.2.0
+python -m virtualenv /home/travis/.venv
AttributeError: 'Path' object has no attribute 'write_bytes'
The command "./.travis/install.sh" failed and exited with 1 during .
```

**Comment by ThomasWaldmann** [comment]
https://github.com/pypa/virtualenv/issues/1963

this seems to be a 20.0.32 regression.

**Comment by ThomasWaldmann** [comment]
this was fixed in virtualenv 20.0.33.
